gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/service/impl/UserServiceImpl.java

```diff
@@ -113,7 +113,7 @@ public class UserServiceImpl implements UserService {
     private GroupService groupService;
 
     @Autowired
-    private RoleService roleService;
+    private RoleFacade roleFacade;
 
```

Comment:
as far as I understand, the SCIM services should remain intact. The new fallback mechanism should handle OIDC flow and token generation

Reply:
correct, RoleFacade shouldn't be used in SCIM.